.github/workflows/cypress.yml

```diff
@@ -0,0 +1,50 @@
+name: Cypress Tests
+
+on: [push]
```

Comment:
are we sure we want to run this workflow every push instead of pull-request?

Reply:
no, pull-request would be more apt. :pray:

Reply:
fix - https://github.com/offtheair/onthechain/pull/3/commits/802e4613d4e68c36220041fc271b2216a9aab858